katalog/node-exporter/deploy.yml

```diff
@@ -16,16 +16,19 @@ spec:
     spec:
       containers:
       - args:
-        - --web.listen-address=0.0.0.0:9100
+        - --web.listen-address=127.0.0.1:9100
         - --path.procfs=/host/proc
         - --path.sysfs=/host/sys
+        - --path.rootfs=/host/root
+        - --no-collector.wifi
+        - --no-collector.hwmon
         - --collector.filesystem.ignored-mount-points=^/(dev|proc|sys|var/lib/docker/.+)($|/)
         - --collector.filesystem.ignored-fs-types=^(autofs|binfmt_misc|cgroup|configfs|debugfs|devpts|devtmpfs|fusectl|hugetlbfs|mqueue|overlay|proc|procfs|pstore|rpc_pipefs|securityfs|sysfs|tracefs)$
-        image: quay.io/prometheus/node-exporter:v0.16.0
+        image: quay.io/prometheus/node-exporter:v0.18.1
         name: node-exporter
         resources:
           limits:
-            cpu: 102m
+            cpu: 250m
             memory: 180Mi
           requests:
             cpu: 102m
```

Comment:
Should we setup the same value for request and limits? I see a CPU Limit increase but CPU request remains in 102m

Reply:
I think it's related to this PR: coreos/prometheus-operator#2144

Ref: kubernetes-monitoring/kubernetes-mixin#108